scripts/wdl/case_gatk_acnv_workflow.wdl

```diff
@@ -8,8 +8,14 @@
 # tumor_entity  tumor_bam  tumor_bam_index  normal_entity  normal_bam  normal_bam_index  <--second input
 # etc...
 #
+# - set isWGS variable to true or false to specify whether to run a WGS or capture workflow respectively
+#
 # - file names will use the entity ID specified, but inside the file, the bam SM tag will typically be used.
 #
+# - target_bed input is only used with capture workflow, WGS workflow generates its own targets (so user can pass any string as an argument)
```

Comment:
Are these still BED files with the latest changes by @davidbenjamin in #582?  If not, change `target_bed` accordingly.